WcaOnRails/app/controllers/polls_controller.rb

```diff
@@ -42,11 +42,11 @@ def edit
   def update
     @poll = Poll.find(params[:id])
     if @poll.update_attributes(poll_params)
-      if params[:commit] == "Confirm"
-        flash[:success] = "Poll confirmed and open to voting"
-      else
-        flash[:success] = "Updated poll"
-      end
+      flash[:success] = if params[:commit] == "Confirm"
+                          "Poll confirmed and open to voting"
+                        else
+                          "Updated poll"
+                        end
```

Comment:
I personally don't like the additional indentation this causes. How do you feel about turning off the cop and letting people follow their hearts with these?